internal/provisioner/registry/main.go

```diff
@@ -179,7 +181,7 @@ func main() {
 	}
 
 	const registryBundleProvisionerName = "registry"
-	unpacker, err := source.NewDefaultUnpacker(mgr, ns, registryBundleProvisionerName, unpackImage)
+	unpacker, err := source.NewDefaultUnpacker(mgr, ns, registryBundleProvisionerName, unpackImage, baseBinaryDownloadURL, rootCAs)
```

Comment:
This may be a candidate for refactoring given the number of function parameters now.

Reply:
Adding a TODO comment. Seems like something better left to a dedicate PR. WDYT?

Reply:
Yeah, that's fine with me. I'm a big fan of smaller PRs anyways.